### PR TITLE
Deliverable 1 - Load Products & UI

### PR DESCRIPTION
**Pull request by RogFed**
- Load products from mock file
- Display Products UI
- Show required info for each product

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`);
+});
+
+app.use(express.static(path.join(__dirname, '..', 'build')));
+
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname, '..', 'build', 'index.html'));
+});
```
Comment (YeipiLozano) [inline server/server.js:14]: would you mind telling me what the server is for? 😮

**Review by YeipiLozano — APPROVED** [review]
Extra points for pro-activity! I see you already added redux and saga, excellent! Just a little heads-up, the final implementation we will make use of redux-toolkit and thunk for async calls. Other than that, great job 👍

File: server/server.js (hunk 1 above)
Comment (RogFed) [reply server/server.js:14]: It's what Heroku uses to run the App. 
https://shielded-savannah-94425.herokuapp.com/

**Comment by RogFed** [comment]
> Extra points for pro-activity! I see you already added redux and saga, excellent! Just a little heads-up, the final implementation we will make use of redux-toolkit and thunk for async calls. Other than that, great job 👍

Thanks JP, I wanted to refresh some of my 'core' redux concepts. I work with it constantly but I rarely have the chance to configure the whole thing. But yes, I will add RTK for the next Deliverables. Thanks for the review

**Pull request merged**